src/utils/utils_common.h

```diff
@@ -29,8 +29,10 @@ extern "C" {
         expression;                                                            \
     } while (0)
 
-#define ALIGN_UP(value, align) (((value) + (align)-1) & ~((align)-1))
-#define ALIGN_DOWN(value, align) ((value) & ~((align)-1))
+#ifndef ALIGN_UP
+#define ALIGN_UP(value, align) (((value) + (align) - 1) & ~((align) - 1))
+#define ALIGN_DOWN(value, align) ((value) & ~((align) - 1))
+#endif
```

Comment:
This looks like random change not related to this PR

Reply:
we define ALIGN_UP in two places - here and in test/common/test_helpers.h